packages/pages-shared/metadata-generator/createMetadataObject.ts

```diff
@@ -61,18 +68,31 @@ function constructRedirects({
 	const num_valid = redirects.rules.length;
 	const num_invalid = redirects.invalid.length;
 
+	// exhaustive check, since we could not have parsed `redirects` out of
+	// a non-existing redirects file
+	const redirectsRelativePath = redirectsFile
+		? relative(process.cwd(), redirectsFile)
+		: "";
```

Comment:
Could this just be an assert on `redirectsFile`?

Reply:
oh, `assert`ing on that redirectsFile actually breaks some `pages-shared` files, so I'd rather leave it out